src/pages/homePage/Home.js

```diff
@@ -1,8 +1,35 @@
 import React from "react";
-import { HomePageContainer } from "./home.styles";
+import {
+  StyledHomePageContainer,
+  StyledSearchPart,
+  StyledSearchPartIcons,
+  StyledSearchPartInput,
+  StyledSearchPartTitle,
+} from "./home.styles";
```

Comment:
change that file name to styles.js, in every folder you can have styles.js and there is no need to have a extension like `.styled`